concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1136,24 +1136,24 @@ Hale-1973-1798-1134	Human_Mind_and_Conditions-22	pain	n	10a.22	1783	PAIN
 Hale-1973-1798-1135	Human_Mind_and_Conditions-23	weeping	n	10a.23		
 Hale-1973-1798-1136	Human_Mind_and_Conditions-24	weakness	n	10a.24		
 Hale-1973-1798-1137	Human_Mind_and_Conditions-25	fever	n	10a.25	1534	FEVER
-Hale-1973-1798-1138	Human_Mind_and_Conditions-26	cough	n	10a.26		
-Hale-1973-1798-1139	Human_Mind_and_Conditions-27	sneeze	n	10a.27		
-Hale-1973-1798-1140	Human_Mind_and_Conditions-28	hiccup	n	10a.28		
+Hale-1973-1798-1138	Human_Mind_and_Conditions-26	cough	n	10a.26	3913	COUGH (ACT OF COUGHING)
+Hale-1973-1798-1139	Human_Mind_and_Conditions-27	sneeze	n	10a.27	1621	SNEEZE
+Hale-1973-1798-1140	Human_Mind_and_Conditions-28	hiccup	n	10a.28	162	HICCOUGH
 Hale-1973-1798-1141	Human_Mind_and_Conditions-29	disease	n	10a.29	1986	DISEASE
 Hale-1973-1798-1142	Human_Mind_and_Conditions-30	leprosy	n	10a.30	3624	LEPROSY
 Hale-1973-1798-1143	Human_Mind_and_Conditions-31	tuberculosis	n	10a.31		
 Hale-1973-1798-1144	Human_Mind_and_Conditions-32	malaria	n	10a.32	3059	MALARIA
 Hale-1973-1798-1145	Human_Mind_and_Conditions-33	cholera	n	10a.33		
 Hale-1973-1798-1146	Human_Mind_and_Conditions-34	typhoid	n	10a.34		
 Hale-1973-1798-1147	Human_Mind_and_Conditions-35	smallpox	n	10a.35	1054	SMALLPOX
-Hale-1973-1798-1148	Human_Mind_and_Conditions-36	measles	n	10a.36		
+Hale-1973-1798-1148	Human_Mind_and_Conditions-36	measles	n	10a.36	3389	MEASLES
 Hale-1973-1798-1149	Human_Mind_and_Conditions-37	cold	n	10a.37	102	COMMON COLD (DISEASE)
 Hale-1973-1798-1150	Human_Mind_and_Conditions-38	eye disease	n	10a.38		
 Hale-1973-1798-1151	Human_Mind_and_Conditions-39	dysentery	n	10a.39		
 Hale-1973-1798-1152	Human_Mind_and_Conditions-40	worms	n	10a.40	1219	WORM
 Hale-1973-1798-1153	Human_Mind_and_Conditions-41	poison	n	10a.41	655	POISON
 Hale-1973-1798-1154	Human_Mind_and_Conditions-42	medicine	n	10a.42	1372	MEDICINE
-Hale-1973-1798-1155	Human_Mind_and_Conditions-43	cure	n	10a.43		
+Hale-1973-1798-1155	Human_Mind_and_Conditions-43	cure	n	10a.43	499	CURE
```

Comment:
`499 CURE` in concepticon is a verb (https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L500), so I would unmap.